cluster-agent/README.md

```diff
@@ -57,20 +94,21 @@ helm install appdynamics-charts/cluster-agent --name=stable \
 | `deployment.image.tag` | Tag of the machine agent image | `latest`
 | `deployment.image.pullPolicy` | The machine agent image pull policy| `IfNotPresent`
 | `deployment.resources ` | Resources directive  | See below
-
+| `existingSecret` | Name of existing secret to use for ClusterAgent application in AppDyanamics   |
 
 Example resource limits:
 
 ```
   resources:
     limits:
       cpu: 200m
-      memory: "300M"
-    requests: 
+      memory: "300Mi"
+    requests:
       cpu: 100m
-      memory: "100M"
+      memory: "100Mi"
  ```
+## Limitation
```

Comment:
Added limitation as I could not make chart work with AppDyanamics Trial account.
Please let me know we need more investigation on using AppDynamics lite account.